<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>554</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>554</width>
    <height>600</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>521</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="saveSelect" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="saveFieldLabel">
         <property name="text">
          <string>Savefile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="saveNameLabel">
         <property name="text">
          <string>Not selected.</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveSelectButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="outputSelect" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="outputFieldLabel">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>output.png</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="outputSelectButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="orientation">
       <item>
        <widget class="QLabel" name="orientationLabel">
         <property name="text">
          <string>Orientation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="orientationNW">
         <property name="text">
          <string>North-West</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="orientationSW">
         <property name="text">
          <string>South-West</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="orientationSE">
         <property name="text">
          <string>South-East</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="orientationNE">
         <property name="text">
          <string>North-East</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="dimensionSelect">
       <item>
        <widget class="QLabel" name="dimensionLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Dimension</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dimensionSelectDropDown">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="xBounds">
       <item>
        <widget class="QLabel" name="minXLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>From X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minX">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="maxXLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>To X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxX">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="zBounds">
       <item>
        <widget class="QLabel" name="minZLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>From Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minZ">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="maxZLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>To Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxZ">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="yBounds">
       <item>
        <widget class="QLabel" name="minYLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>From Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minY">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="maxYLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>To Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxY">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="padding">
       <item>
        <widget class="QLabel" name="paddingFieldLabel">
         <property name="text">
          <string>Image Padding</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="paddingValue">
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="misc">
       <item>
        <widget class="QCheckBox" name="shading">
         <property name="text">
          <string>Shading</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lighting">
         <property name="text">
          <string>Lighting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hideBeacons">
         <property name="text">
          <string>Hide Beacons</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hideWater">
         <property name="text">
          <string>Hide Water</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="colorSelect" stretch="0,1,0,0">
       <item>
        <widget class="QLabel" name="colorFieldLabel">
         <property name="text">
          <string>Colorfile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="colorFileLabel">
         <property name="text">
          <string>Not selected.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorSelectButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorResetButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renderButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Render</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionToggleLogs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionToggleLogs">
   <property name="text">
    <string>Show logs</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
 </widget>
</ui>
